db/migrate/20231026032737_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
```

Comment:
- [ ] [Optional] [Highly recommended] Same here for `:commentsCounter` and `:likesCounter`, Using the snake case for naming columns inside rails would be better professionally and considered a good practice. Please name them `:comments_counter` and `:likes_counter` instead.